port_ocean/helpers/retry.py

```diff
@@ -55,14 +55,14 @@ class RetryTransport(httpx.AsyncBaseTransport, httpx.BaseTransport):
             HTTPStatus.GATEWAY_TIMEOUT,
         ]
     )
-    MAX_BACKOFF_WAIT = 60
+    MAX_BACKOFF_WAIT = 60 * 5  # 5 minutes
```

Comment:
changing this here will effect both port client and all integrations clients, is it intended?